desktop/apps/article/templates/classic_meta.jade

```diff
@@ -45,8 +45,8 @@ if article.get('published')
       meta( name="sailthru.credit_line" content=article.get('email_metadata').credit_line)
     if article.get('email_metadata').image_url
       - var src = article.get('email_metadata').image_url
-      meta( name="sailthru.image.full" content=crop(src, { width: 1200, height: 706 } ))
-      meta( name="sailthru.image.thumb" content=crop(src, { width: 900, height: 530 } ))
+      meta( name="sailthru.image.full" content=crop(src, { width: 1200, height: 800 } ))
```

Comment:
🤔  I wonder if there're ways to dynamically change these. e.g. Does Sailthru send dimension information in the url or headers? Not blocking this PR for it as I know this is time sensitive.

Reply:
Hm interesting! If we wanted to make this dynamic, we'd probably have to put the responsibility of wrapping an image with on-the-fly resizing on the Sailthru template and pass in `src` without `crop`. All of this ST metadata gets used directly as content in emails. 

I wonder if the emails team is using on-the-fly in their templates already (I'll ask). Although it's a minor version of it, it does bring up that question around data and whether we send raw stuff or cooked content.